config/webpack.config.js

```diff
@@ -34,7 +34,7 @@ const configGenerator = (options) => {
     filesToBuild = _.pick(entryFiles, options.entry.split(',').map(x => x.trim()));
   }
   filesToBuild.vendor = [
-    'core-js',
+    './src/js/common/polyfills',
```

Comment:
If we leave core-js here, then we'll always bundle all of core-js in the vendor chunk, which defeats the purpose of having the Babel transform only include the polyfills we need.